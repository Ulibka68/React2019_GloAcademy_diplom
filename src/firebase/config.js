
  const firebaseConfig = {
    apiKey: process.env.REACT_APP_API_KEY,
    authDomain: "vlasov-react-gloacademy-diplom.firebaseapp.com",
    databaseURL: "https://vlasov-react-gloacademy-diplom.firebaseio.com",
    projectId: "vlasov-react-gloacademy-diplom",
    storageBucket: "vlasov-react-gloacademy-diplom.appspot.com",
    messagingSenderId: process.env.REACT_APP_MESSAGINGSENDERID,
    appId: process.env.REACT_APP_APPID
  };




  export default firebaseConfig;