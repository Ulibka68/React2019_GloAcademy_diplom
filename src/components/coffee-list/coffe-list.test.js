import React from "react";
import {shallow,mount} from "enzyme";
import CoffeeList from "./coffe-list";
import config from "../../services/setupTests";
import { MemoryRouter, Route } from 'react-router-dom';
import renderer from 'react-test-renderer';

const props = {
    filtered : false,
    filteredButton : "",
    filteredString : ""
};

describe('<CoffeeFilteredList render />', () => {
    it('renders correctly', () => {
        const list = mount(
                <CoffeeList {...props} />
            );
          
        expect(list.debug()).toMatchSnapshot();
        
      });

});  

describe('<CoffeeFilteredList render />', () => {
    it('renders correctly', () => {
        const list = shallow(
                <CoffeeList {...props} />
            );
          
            expect(list.state('isLoading')).toBe(true);
      });

});  