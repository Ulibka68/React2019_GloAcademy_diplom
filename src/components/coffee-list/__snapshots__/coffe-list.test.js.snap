// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoffeeFilteredList render /> renders correctly 1`] = `
"<CoffeeList filtered={false} filteredButton=\\"\\" filteredString=\\"\\">
  <Spinner>
    <div>
      <div className=\\"cssload-loading\\">
        loading ...
      </div>
    </div>
  </Spinner>
</CoffeeList>"
`;
